notification-service/src/main/resources/application-local.yml

```diff
@@ -43,9 +43,10 @@ spring:
     username: postgres
     password: postgres
 
-springdoc:
-  writer-with-order-by-keys: true
-  writer-with-default-pretty-printer: true
```

Comment:
These are necessary to properly generate SDK files. Why do you delete them?

Reply:
These properties are redundant, they are in application.yml so for local profile it will be inherited from the https://github.com/parodos-dev/parodos/blob/main/notification-service/src/main/resources/application.yml

Reply:
You're right! Thanks